### PR TITLE
Lps 120269

### PR DESCRIPTION
**Pull request by arthurchan35**

### CODE REVIEW

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:sf

**Comment by csierra** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by csierra** [comment]
ci:test:sf

**Comment by csierra** [comment]
ci:test:relevant

**Comment by csierra** [comment]
/cc @Preston-Crary @shuyangzhou

hi... could you folks give us a hand with this one?
we are just moving it from portal to here but I don't know what's missing here.

Thx in advance.

Carlos.

**Comment by Preston-Crary** [comment]
@csierra you need to update the Eclipse, NetBeans and Intellij project files. Also need to update the [jboss module.xml](https://github.com/liferay/liferay-portal/blob/master/tools/servers/jboss/modules/module.xml). You can use one of the tickets I did as an example to make sure there's nothing else that's needed.

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:sf

File: .classpath [hunk 1]
```diff
@@ -18,6 +18,7 @@
 	<classpathentry excluding="**/.svn/**|.svn/" kind="src" path="modules/core/petra/petra-sql-dsl-api/src/main/java"/>
 	<classpathentry excluding="**/.svn/**|.svn/" kind="src" path="modules/core/petra/petra-sql-dsl-spi/src/main/java"/>
 	<classpathentry excluding="**/.svn/**|.svn/" kind="src" path="modules/core/petra/petra-string/src/main/java"/>
+	<classpathentry excluding="**/.svn/**|.svn/" kind="src" path="modules/core/petra/petra-pattern-mapper/src/main/java"/>
```
Comment (csierra) [inline .classpath:21]: hey @arthurchan35, I believe there is a typo here, isn't there?

**Comment by csierra** [comment]
> @csierra you need to update the [Eclipse](https://github.com/liferay/liferay-portal/blob/master/.classpath), [NetBeans](https://github.com/liferay/liferay-portal/tree/master/nbproject) and Intellij project files (if used in portal-impl/portal-kernel). Also need to update the [jboss module.xml](https://github.com/liferay/liferay-portal/blob/master/tools/servers/jboss/modules/module.xml). You can use one of the tickets I did as an example to make sure there's nothing else that's needed.

thanks @Preston-Crary!

File: .classpath (hunk 1 above)
Comment (arthurchan35) [reply .classpath:21]: @csierra ,

Thanks Carlos, I looked at them after commiting, but still couldnt see this one..

**Comment by arthurchan35** [comment]
ci:stop

File: .classpath (hunk 1 above)
Comment (csierra) [reply .classpath:21]: sorry to be the bearer of bad news... :sweat:

**Comment by arthurchan35** [comment]
ci:test:relevant

**Comment by arthurchan35** [comment]
ci:test:sf

**Comment by csierra** [comment]
hi again @Preston-Crary @shuyangzhou

it looks we are still missing something... could you please point us to one of those tickets you mention or to what is missing?

Thanks!

Carlos.

**Comment by arthurchan35** [comment]
see:
https://github.com/brianchandotcom/liferay-portal/pull/94590

**Pull request closed without merging**